apps/examples/src/examples/api/README.md

```diff
@@ -1,7 +1,8 @@
 ---
 title: Editor API
 component: ./APIExample.tsx
-order: 2
+category: editor
+order: 1
```

Comment:
did i miss where these `order`s are being used? maybe you pulled it out later?

in any case, it's a moot point: we shouldn't have `order` - if you pull an example or add a new one, you would have to change all the numbers in several files - no fun :P

we can talk about different approaches on this but, either we have maybe a `priority` option where some things bubble up higher or lower in the list, or, i guess, alphabetize.

Reply:
I forgot to sort them! I was going to order them all according to their level of complexity, but you raise a good point about maintainability. I think I'll go for priority, a number between 1 and 3 that communicates how complex or difficult the example is.